bench/workgen/runner/example_prepare.py

```diff
@@ -85,6 +89,6 @@
 
 print('Workload took %d minutes' %(run_time//60))
 
-latency_filename = homedir + "/latency.out"
+latency_filename = "WT_TEST" + "/latency.out"
```

Comment:
I think I prefer keeping the variable `homedir` here, that way the home directory can be changed in one place.

Reply:
Fixed.